zebra-rpc/src/methods/types/get_blockchain_info.rs

```diff
@@ -82,9 +82,9 @@ impl GetBlockchainInfoBalance {
         Self::new_internal("orchard", amount, delta)
     }
 
-    /// Creates a [`GetBlockchainInfoBalance`] for the Deferred pool.
-    pub fn deferred(amount: Amount<NonNegative>, delta: Option<Amount<NegativeAllowed>>) -> Self {
-        Self::new_internal("deferred", amount, delta)
+    /// Creates a [`GetBlockchainInfoBalance`] for the Lockbox pool.
+    pub fn lockbox(amount: Amount<NonNegative>, delta: Option<Amount<NegativeAllowed>>) -> Self {
```

Comment:
This is a public function that we don't need to break. I prefer to keep the `deferred` function name here or add this breaking change to the CHANGELOG as well.